### ISSUE TITLE
Engine prime too difficult at cold weather

### ISSUE THREAD

**Issue by HHS81** [HHS81/c182s#249]
Engine prime is hard below -3°C, and impossible below -6°C without preheating.

According POH p. 4-33 the procedure is the same as warm temperatures. There might be only frozen sparks, which needs a preheater below -6°C before restart.

Also some pilot discussion on the web states, that it should be easy to start a fuel-injected engine at cold weather.

Lycoming says: "For coldweather starting, it may be necessary to keep the mixture control in rich somewhat longer than in warm weather."  --> Air more dense - more fuel needed.

Suggestion: 
keep it more easy until -6°C or lower (-10°C or -15°C  ??) and then make it impossible at much lower temperatures (like -25°C).
You still need a preheater below -6°C, as this might make starts more easy. We might consider simulating frozen sparks 

Something like:

```
<fcs_function name="/systems/fuel/priming-temperature-threshold">
            <function>
                <table>
                    <independentVar lookup="row">/engines/engine/oil-final-temperature-degf</independentVar>
                    <tableData>
                        <!-- info: accoring to POH (4-33) one should use a external heater below 20Â°F / -6Â°C) -->
                        -13.0   75.0   <!-- -25°C --> <!-- flooded-point is 75%, so below this temp no start is possible without external heat! -->
                        -5    68.0   <!-- below -15°C it is very hard to prime -->
                        32.0    56.0   <!--  0°C -->
                        50.0    42.5   <!-- 10°C --> <!-- a little more than 3s needed -->
                        68.0    20.0   <!-- 20°C --> <!-- a little less than 2s needed -->
                        75.0     0.0   <!-- 24°C --> <!-- engine is considered "hot" and dont need priming - if there is still fuel in the manifold -->
                        86.0     0.0   <!-- 30°C -->
                    </tableData>
                </table>
            </function>
            <output>/systems/fuel/priming-temperature-threshold</output>   
        </fcs_function>
```

Link: [https://www.lycoming.com/content/operating-cold-weather](https://www.lycoming.com/content/operating-cold-weather)

**Comment by hbeni** [comment]
What i also have read is that with colder temps the oil might be coagled and adds much friction.
In the current implementation i tought that heat directly correlates to the spark-ability, but that doesnt seem to be the case.

**Comment by hbeni** [comment]
Your suggested change seems plausible :)

**Comment by HHS81** [comment]
@hbeni 
Viscosity increases with colder temperatures. Ron Jensens method is able to simulate this, as I can remember

**Comment by hbeni** [comment]
fixed with d2dd718d41fc
